app/models/Search.java

```diff
@@ -263,14 +263,18 @@ public boolean doingScrollScanNow() {
 		return doingScrollScanNow;
 	}
 
-	private void bulk(final Request request, final Serialization serialization) {
+	private void bulk(final Request request, final Serialization serialization,
+			final String scroll) {
 		boolean JSON_LD = serialization.equals(Serialization.JSON_LD);
+		boolean RDF_XML = serialization.equals(Serialization.RDF_XML);
 		try {
 			long lastTime = Calendar.getInstance().getTimeInMillis();
-			SearchResponse searchResponse = startInitialResponse();
+			SearchResponse searchResponse = startInitialResponse(scroll);
 			final SearchHits hits = getTotalHits(searchResponse);
 			if (JSON_LD)
 				getMessageOut().write("[");
+			if (RDF_XML)
+				getMessageOut().write("<root>");
```

Comment:
You're right, with `scroll` the response in production is not well-formed, and the wrapped `<root>` works fine with rapper.